ES/ESmy022.xml

```diff
@@ -30,226 +30,359 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                          </objectDesc>
+
+                                      <handDesc>
+                                         <handNote xml:id="h2" script="Ethiopic">
+                                            <seg type="script">Mid-nineteenth/mid-twentieth century bulky script.</seg>
+                                            <seg type="ink">Black (on many folia brownish); vivid red.</seg>
+                                            <seg type="rubrication">Divine names; directives for the ministers;
+                                              titles of books to be read; elements of the punctuation signs;
+                                               elements of the Ethiopic numerals. On the incipit page of <ref target="#p1_i1">Text 1</ref>, <locus target="#8r"/>
+                                               two first lines of each column are rubricated. Rubrication has been carefully executed in the main
+                                               hand, probably along with the writing of the main text.</seg>
+                                               <desc>Careful but irregular; by a poorly trained scribe. Broad, upright or slightly
+                                                 right sloping script. Downstrokes slanted irregulary, horizontal tops.
+                                                 Rectangular letter shapes, but irregular and poorly drawn.
+                                              Starting from ca. <locus target="#47"/>, the handwriting becomes even less careful;
+                                              on <locus from="50r" to="51r"/> the scribe reduced the size of the script
+                                              and accomodated on the pages
+                                              more written lines than the ruling allows.
+                                              It cannot be excluded that more than one scribe participated in copying.
+                                               </desc>
+                                         </handNote>
+                                      </handDesc>
+
+
+                                      <decoDesc>
+                                        <decoNote type="frame" xml:id="d1">
+                                           <locus target="#8r"/>:
+                                           <desc>Crude bichrome (black, red) ornamental
+                                           band.</desc>
+                                         </decoNote>
+
+                                         <decoNote type="drawing" xml:id="d2">
+                                            <locus target="#52v"/>
+                                            <desc>A figure of an equestrian saint, crudely drawn in pencil, poorly visible.</desc>
+                                         </decoNote>
+                                       </decoDesc>
+
+                                      <additions>
```

Comment:
I would assume that additions, unless you know something I do not know, belong not to this msPart but to the entire MS and should be moved up, before the first msPart is introduced - they may have been added after the two parts were united

Reply:
In this case, the notes belong most probably to part 1, the core.